### DISCUSSION TITLE
GitHub Pages não direciona o usuário para a página correta

### DISCUSSION THREAD

**Discussion by CristianLF23** [community/community#29628] [New to GitHub] [answered]
Olá! Estou tendo problemas com o Pages no site em HTML, CSS e JS que dei commit. A página principal, o index.html é exibida corretamente (https://cristianlf23.github.io/Pagina-de-cadastro/), ao preencher o formulário corretamente, o usuário deveria receber uma página de cadastro concluído, porém, ao invés de direcionar para o endereço correto (https://cristianlf23.github.io/Pagina-de-cadastro/concluido.html) ele redireciona para outro que está incorreto e dá o erro 404. Como posso resolver isso? O redirecionamento de página está sendo feito através do JavaScript no meu código. Link do meu repositório: https://github.com/CristianLF23/Pagina-de-cadastro

**Comment by mazkdevf** [comment, marked as answer]
Hello Cristian!

https://github.com/CristianLF23/Pagina-de-cadastro/pull/1

**Reply by mazkdevf** [reply]
Check that out

**Reply by CristianLF23** [reply]
It worked! I didn't thougt it was something so simple hahaha thanks!!!!

**Reply by mazkdevf** [reply]
👍🏻 np, have a good day!
